### PR TITLE
Fix/metanetx parsing

### PR DESCRIPTION
**Pull request by phantomas1234**
The different universal models were previously based on the source
column of the reac_prop table from metanetx. Unfortunately, this
did not reflect the actual reaction content of the different databases
(bigg, rhea, ...). Now the universal models are based on the reac_xref
mapping.

### CODE REVIEW

File: scripts/parse_metanetx.py [hunk 1]
```diff
@@ -64,13 +64,14 @@ def parse_coeff_and_metabolites(term):
     return stoichiometry
 
 
-def construct_universal_model(list_of_db_prefixes):
+def construct_universal_model(list_of_db_prefixes, reac_xref, reac_prop, chem_prop):
```
Comment (hredestig) [inline scripts/parse_metanetx.py]: perhaps it could afford a docstring as not immediately obvious what `reac_xref` should be..

**Comment by matthiaskoenig** [comment]
Just a question: How did you get the metanetx database? Is there some download somewhere/or some scrapping script which queries the whole database?

**Comment by Midnighter** [comment]
@matthiaskoenig: http://www.metanetx.org/mnxdoc/mnxref.html

**Comment by phantomas1234** [comment]
Closing this in favor of integrating solution into old cameo

**Pull request closed without merging**